css-view-transitions-1/Overview.bs

```diff
@@ -1158,104 +1237,209 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 				Issue: This needs proper types.
 
 			1. Set |namedElements|[|transitionName|] to |capture|.
+	</div>
 
-		1. Set |document|'s [=document/transition suppressing rendering=] to true.
+### [=Capture the new state=] ### {#capture-new-state-algorithm}
```

Comment:
This is just extracted from the "Setup view transition" algorithm. No changes other than returning failure, rather than throwing an error.